modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/memodelrelease/v0.1.1.json

```diff
@@ -18,7 +18,7 @@
       "nodekind": "sh:BlankNodeOrIRI",
       "and": [
         {
-          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+          "node": "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1/shapes/ModelInstanceShape"
```

Comment:
Is a MEModelReleaseShape a Model ? Only models can extends the ModelInstanceShape.

Reply:
Yes, MEModelReleases need to be annotated with brain region and species information.